hbase_regionserver/conf.yaml.example

```diff
@@ -0,0 +1,530 @@
+instances:
+  - host: localhost
+    port: 10102 # This is the JMX port on which HBase RegionServer exposes its metrics (usually 10101)
+    tags:
+      hbase: regionserver
+      # env: stage
+      # newTag: test
+    # user: username
+    # password: password
+    # process_name_regex: .*process_name.* # Instead of specifying a host and port or jmx_url, the agent can connect using the attach api.
+    #                                      # This requires the JDK to be installed and the path to tools.jar to be set below.
+    # tools_jar_path: /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar # To be set when process_name_regex is set
+    # name: hbase_master
+    # # java_bin_path: /path/to/java # Optional, should be set if the agent cannot find your java executable
+    # # java_options: "-Xmx200m -Xms50m" # Optional, Java JVM options
+    # # trust_store_path: /path/to/trustStore.jks # Optional, should be set if ssl is enabled
+    # # trust_store_password: password
+
+init_config:
+  is_jmx: true
+
+  # Metrics collected by this check. You should not have to modify this.
```

Comment:
Could you add comment for each attribute so people know what they are collecting and why it's interesting ?
Or at least the main ones.